clar2wasm/src/words/noop.rs

```diff
@@ -0,0 +1,116 @@
+use crate::wasm_generator::{GeneratorError, WasmGenerator};
+use clarity::vm::{ClarityName, SymbolicExpression};
+
+use super::Word;
+
+fn traverse_noop(
```

Comment:
Done

Reply:
> (in case of to-int and to-uint)

This is actually true of all three. `contract-of` is also just changing the type from a trait to a principal, which are both represented the same way.